### PR TITLE
update(HTML): web/html/element/video

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [&lt;video&gt; – елемент вбудованого відео@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/video), [сирці &lt;video&gt; – елемент вбудованого відео@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/element/video/index.md)

Нові зміни:
- [Remove self links, other pages (#35041)](https://github.com/mdn/content/commit/9231a7046973685f4600e1891fa644ecce41ef3b)
- [Convert noteblocks for web/html folder (#35114)](https://github.com/mdn/content/commit/9c09b183a5ce844a75c2f22e909d03f71ca329fc)
- [Fix code-link containing spaces, part 1 (#35065)](https://github.com/mdn/content/commit/998a4e6fc713678381e6bc31130e3a0a53158d69)
- [video element typo (#34832)](https://github.com/mdn/content/commit/944a361f159066f48a056622ed7c76c6e9b17b03)
- [WebVTT - general restructure to match normal API docs (#34034)](https://github.com/mdn/content/commit/3975bcf6caa09c9c5f7fddf2eef2be6c021d00f6)
- [fix(html): remove ogv file from example (#34514)](https://github.com/mdn/content/commit/282fc6460418efa5a8da90a5900b98bfede72ff0)
- [Update note about autoplaying audio (#34569)](https://github.com/mdn/content/commit/f272744d917e9604bad30193b201bdc927e5a7d4)
- [HTML element attr list should link to attr page (#34523)](https://github.com/mdn/content/commit/991385e7cfb9ac8589332b07aadcc4b38edea512)
- [Move Accessibility section above Examples for html element pages (#34007)](https://github.com/mdn/content/commit/bde0cb215d1d307c08678abe6623fc0d39f4cf7f)

### CODE REVIEW

File: files/uk/web/html/element/video/index.md [hunk 1]
```diff
@@ -296,16 +294,37 @@ browser-compat: html.elements.video
 
 Особливих міркувань щодо стилізації `<video>` немає; загальною стратегією є надати йому властивість `display` зі значенням `block`, щоб йому було легше задати розташування, розмір тощо, а потім надати стилізацію та інформацію про компонування, як необхідно. Певні методи стилізації наведено в [основах стилізації програвача відео](/uk/docs/Web/Media/Audio_and_video_delivery/Video_player_styling_basics).
 
+### Додавання титрів та інших хронометрованих текстових доріжок
+
+Хронометровані текстові доріжки з титрами, закритими субтитрами, заголовками розділів і так далі можна декларативно додавати, вкладаючи їх у елемент {{HTMLElement("track")}}.
```
Comment (undead404) [inline files/uk/web/html/element/video/index.md:299]:
```suggestion
Хронометровані текстові доріжки з титрами, закритими субтитрами, заголовками розділів і так далі можна декларативно додавати, вкладаючи їх в елемент {{HTMLElement("track")}}.
```

File: files/uk/web/html/element/video/index.md [hunk 2]
```diff
@@ -296,16 +294,37 @@ browser-compat: html.elements.video
...
-  - : {{domxref("AudioTrackList")}}, що містить усі аудіодоріжки мультимедійного елемента. До цього об'єкта можна додати слухача `addtrack`, щоб отримати сповіщення, коли до елемента додаються нові аудіодоріжки.
+  - : {{domxref("AudioTrackList")}}, що містить усі аудіодоріжки мультимедійного елемента.
+    Додайте до цього об'єкта слухач `addtrack`, щоб отримати сповіщення, коли до елемента будуть додані нові аудіодоріжки.
 - {{domxref("HTMLMediaElement.videoTracks")}}
-  - : Додайте слухача `addtrack` до цього об'єкта {{domxref("VideoTrackList")}}, щоб отримати сповіщення, коли до елемента додаються нові доріжки відео.
+  - : A {{domxref("VideoTrackList")}}, що вміщає усі відеодоріжки медійного елемента.
+    Додайте до цього об'єкта слухач `addtrack`, щоб отримати сповіщення, коли до елемента будуть додані нові відеодоріжки.
 - {{domxref("HTMLMediaElement.textTracks")}}
-  - : Додайте слухача `addtrack` до цього {{domxref("TextTrackList")}}, щоб отримати сповіщення, коли до елемента додаються нові текстові доріжки.
+  - : {{domxref("TextTrackList")}}, що вміщає усі текстові доріжки медійного елемента (які використовуються як титри, закриті субтитри і так далі).
```
Comment (undead404) [inline files/uk/web/html/element/video/index.md:326]:
```suggestion
  - : {{domxref("TextTrackList")}}, що вміщає усі текстові доріжки медійного елемента (які використовуються як титри, закриті субтитри й так далі).
```

File: files/uk/web/html/element/video/index.md [hunk 3]
```diff
@@ -23,12 +23,11 @@ browser-compat: html.elements.video
 
   - : Булів атрибут; коли вказаний, то відео автоматично починає відтворення, щойно може, не зупиняючи закінчення завантаження даних
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:24]:
```suggestion
  - : Булів атрибут; коли вказаний, то відео автоматично починає відтворення, щойно може зробити це без зупинки для закінчення завантаження даних.
```

File: files/uk/web/html/element/video/index.md [hunk 4]
```diff
@@ -63,7 +62,7 @@ browser-compat: html.elements.video
 - `loop`
   - : Булів атрибут; коли вказаний, браузер автоматично перемотуватиме на початок після досягнення кінця відео.
 - `muted`
-  - : Булів атрибут, що вказує усталене налаштування для аудіо, включеного у відео. Якщо встановлений, то аудіо спершу буде заглушене. Усталене значення – `false`, отже – аудіо буде відтворено разом з відтворенням відео.
+  - : Булів атрибут, що вказує усталене налаштування заглушення для аудіо, включеного у відео. Якщо встановлений, то аудіо спершу буде заглушене. Усталене значення – `false`, отже – аудіо буде відтворено разом з відтворенням відео.
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:65]:
```suggestion
  - : Булів атрибут, що вказує усталене налаштування заглушення для аудіо, включеного у відео. Якщо встановлений, то аудіо спершу буде заглушене. Усталене значення – `false`, тому аудіо буде відтворено разом з відтворенням відео.
```

File: files/uk/web/html/element/video/index.md [hunk 5]
```diff
@@ -414,79 +456,31 @@ AddType video/webm .webm
 
 {{EmbedLiveSample('dekilka-dzherel', '', '400')}}
 
-Спершу виконується спроба відтворити [Ogg](/uk/docs/Web/Media/Formats/Containers#ogg). Якщо це не вдається, наступна спроба – AVI. Врешті решт, робиться спроба відтворити [MP4](/uk/docs/Web/Media/Formats/Containers#mpeg-4_mp4). Запасне повідомлення виводиться, якщо елемент video не підтримується, але не тоді, коли всі джерела не спрацьовують.
+Спершу виконується спроба відтворити AVI. Якщо це не вдається, наступна спроба [MP4](/uk/docs/Web/Media/Formats/Containers#mpeg-4_mp4). Запасне повідомлення виводиться, якщо елемент video не підтримується, але не тоді, коли всі джерела не спрацьовують.
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:459]:
```suggestion
Спершу виконується спроба відтворити AVI. Якщо це не вдається, то [MP4](/uk/docs/Web/Media/Formats/Containers#mpeg-4_mp4). Запасне повідомлення виводиться, якщо елемент video не підтримується, але не тоді, коли всі джерела не спрацьовують.
```

File: files/uk/web/html/element/video/index.md [hunk 6]
```diff
@@ -296,16 +295,37 @@ browser-compat: html.elements.video
...
+  <track default kind="captions" src="captions.vtt" />
+</video>
+```
+
+Хронометровані текстові доріжки також можна додавати програмно, користуючись [API WebVTT](/uk/docs/Web/API/WebVTT_API).
+
 ### Відстеження додавання й вилучення доріжок
 
-Додавання й видалення з елемента `<video>` доріжок можна відстежувати за допомогою подій {{domxref("VideoTrackList/addtrack_event", "addtrack")}} і {{domxref("VideoTrackList/removetrack_event", "removetrack")}}. Втім, ці події не надсилаються до самого елемента `<video>`. Натомість вони надсилаються до об'єкта списку доріжок всередині {{domxref("HTMLMediaElement")}} елемента `<video>`, що відповідає типові доріжки, котра була додана:
+Додавання й видалення з елемента `<video>` доріжок можна відстежувати за допомогою подій {{domxref("VideoTrackList/addtrack_event", "addtrack")}} і {{domxref("VideoTrackList/removetrack_event", "removetrack")}}. Втім, ці події не надсилаються до самого елемента `<video>`.
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:316]:
```suggestion
Додавання й видалення доріжок з елемента `<video>` можна відстежувати за допомогою подій {{domxref("VideoTrackList/addtrack_event", "addtrack")}} і {{domxref("VideoTrackList/removetrack_event", "removetrack")}}. Втім, ці події не надсилаються до самого елемента `<video>`.
```

File: files/uk/web/html/element/video/index.md [hunk 7]
```diff
@@ -296,16 +295,37 @@ browser-compat: html.elements.video
...
+
+Натомість вони надсилаються до об'єкта списку доріжок всередині {{domxref("HTMLMediaElement")}} елемента `<video>`, що відповідає типові доріжки, котра була додана:
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:318]:
```suggestion
Натомість вони надсилаються до об'єкта списку доріжок всередині {{domxref("HTMLMediaElement")}} елемента `<video>`, що відповідає типу доріжки, котра була додана:
```

File: files/uk/web/html/element/video/index.md [hunk 8]
```diff
@@ -23,12 +23,11 @@ browser-compat: html.elements.video
 
   - : Булів атрибут; коли вказаний, то відео автоматично починає відтворення, щойно може, не зупиняючи закінчення завантаження даних
 
-    > **Примітка:** Сайти, що автоматично відтворюють аудіо (чи відео з аудіостежиною) можуть бути неприємними для користувачів, тому цього слід, за можливості, уникати. Якщо є необхідність запропонувати функціональність автоматичного відтворення, слід зробити її опційною (такою, що вимагає, аби користувач сам її ввімкнув). Проте іще ця функціональність може бути корисною при створенні мультимедійних елементів, чиє джерело буде задане пізніше, під контролем користувача. Дивіться наші [настанови з автоматичного відтворення](/uk/docs/Web/Media/Autoplay_guide), щоб отримати додаткову інформацію про те, як варто використовувати автоматичне відтворення.
+    > [!NOTE]
+    > Сучасні браузери блокують автоматичне відтворення аудіо (та відео з незаглушеною аудіостежиною), адже сайти, що автоматично відтворюють аудіо, можуть бути неприємними для користувачів. Дивіться додаткову інформацію про те, як правильно користуватися автоматичним відтворенням, у нашому [посібнику з автоматичного відтворення](/uk/docs/Web/Media/Autoplay_guide)
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:27]:
```suggestion
    > Сучасні браузери блокують автоматичне відтворення аудіо (та відео з незаглушеною аудіодоріжкою), адже сайти, що автоматично відтворюють аудіо, можуть бути неприємними для користувачів. Дивіться додаткову інформацію про те, як правильно користуватися автоматичним відтворенням, у нашому [посібнику з автоматичного відтворення](/uk/docs/Web/Media/Autoplay_guide)
```

File: files/uk/web/html/element/video/index.md [hunk 9]
```diff
@@ -414,79 +456,31 @@ AddType video/webm .webm
...
     <tr>
       <th scope="row">Дозволений вміст</th>
       <td>
         <p>
           Якщо елемент має атрибут <a href="#src"><code>src</code></a>:
-          нуль чи більше елементів {{HTMLElement("track")}},
-          після котрих – прозорий вміст, що не містить мультимедійних елементів, тобто
-          жодних {{HTMLElement("audio")}} чи
-          {{HTMLElement("video")}}.
+          нуль чи більше елементів {{HTMLElement("track")}}, після котрих – прозорий вміст, що не містить мультимедійних елементів, тобто жодних {{HTMLElement("audio")}} чи <code>&lt;video&gt;</code>.
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:480]:
```suggestion
          нуль чи більше елементів {{HTMLElement("track")}}, за якими прозорий вміст, що не містить мультимедійних елементів, тобто жодних {{HTMLElement("audio")}} чи <code>&lt;video&gt;</code>.
```

File: files/uk/web/html/element/video/index.md [hunk 10]
```diff
@@ -414,79 +456,31 @@ AddType video/webm .webm
...
         </p>
         <p>
-          Інакше: нуль чи більше елементів {{HTMLElement("source")}}, після котрих –
-          нуль чи більше елементів {{HTMLElement("track")}}, після котрих –
-          прозорий вміст, що не містить мультимедійних елементів, тобто жодних
-          {{HTMLElement("audio")}} чи {{HTMLElement("video")}}.
+          Інакше: нуль чи більше елементів {{HTMLElement("source")}}, після котрих – нуль чи більше елементів {{HTMLElement("track")}}, після котрих – прозорий вміст, що не містить мультимедійних елементів, тобто жодних {{HTMLElement("audio")}} чи <code>&lt;video&gt;</code>.
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:483]:
```suggestion
          Інакше: нуль чи більше елементів {{HTMLElement("source")}}, за якими нуль чи більше елементів {{HTMLElement("track")}}, за якими прозорий вміст, що не містить мультимедійних елементів, тобто жодних {{HTMLElement("audio")}} чи <code>&lt;video&gt;</code>.
```

File: files/uk/web/html/element/video/index.md [hunk 11]
```diff
@@ -23,12 +23,11 @@ browser-compat: html.elements.video
 
   - : Булів атрибут; коли вказаний, то відео автоматично починає відтворення, щойно може, не зупиняючи закінчення завантаження даних
 
-    > **Примітка:** Сайти, що автоматично відтворюють аудіо (чи відео з аудіостежиною) можуть бути неприємними для користувачів, тому цього слід, за можливості, уникати. Якщо є необхідність запропонувати функціональність автоматичного відтворення, слід зробити її опційною (такою, що вимагає, аби користувач сам її ввімкнув). Проте іще ця функціональність може бути корисною при створенні мультимедійних елементів, чиє джерело буде задане пізніше, під контролем користувача. Дивіться наші [настанови з автоматичного відтворення](/uk/docs/Web/Media/Autoplay_guide), щоб отримати додаткову інформацію про те, як варто використовувати автоматичне відтворення.
+    > [!NOTE]
```
Comment (alinkedd) [inline files/uk/web/html/element/video/index.md:26]: на 81 рядку ще

**Review by alinkedd — CHANGES_REQUESTED** [review]

**Review by alinkedd — APPROVED** [review]

**Pull request merged**